applications/tari_dan_wallet_daemon/src/jrpc_server.rs

```diff
@@ -104,6 +104,7 @@ async fn handler(
             "get" => call_handler(context, value, token, transaction::handle_get).await,
             "get_result" => call_handler(context, value, token, transaction::handle_get_result).await,
             "wait_result" => call_handler(context, value, token, transaction::handle_wait_result).await,
+            "get_all_by_status" => call_handler(context, value, token, transaction::handle_get_all_by_status).await,
```

Comment:
Could we rename it to `get_all_txs_by_status`? Or `get_all_transacations_by_status`.

Reply:
it's a `transacation.get_all_by_status` command, that sounds redundant to me and also not consistent with other transactions commands.